Cargo.toml

```diff
@@ -34,7 +34,8 @@ tokio = { version = "1.0", features = ["io-std", "macros", "net", "rt-multi-thre
 
 [features]
 default = ["native-tokio", "http1", "tls12", "logging", "ring"]
-aws-lc-rs = ["rustls/aws_lc_rs"]
+aws_lc_rs = ["rustls/aws_lc_rs"]
```

Comment:
Given that (a) this crate was already using the dash-based name and (b) this is a semver-incompatible release, I feel like we should either (1) do the setup the other way around and make `aws_lc_rs` the alias or (2) skip the alias and just require the hyphen-based feature name.

(Perhaps this depends on the question if we want to get rid of the underscore-based feature name in rustls 0.24.)

Reply:
I removed the alias in favour of requiring the hyphen-based feature name for now.